src/main/controllers/CreateThirdPartyUserSuccessController.ts

```diff
@@ -10,13 +10,6 @@ export default class CreateThirdPartyUserSuccessController {
             formData,
         });
 
-        CreateThirdPartyUserSuccessController.removeThirdPartyUserFromFormCookie(res, formData);
-    }
-
-    private static removeThirdPartyUserFromFormCookie(res, formData) {
-        formData.thirdPartyName = '';
-        formData.thirdPartyRole = '';
-        formData.thirdPartyRoleObject = null;
-        res.cookie('formCookie', JSON.stringify(formData), { secure: true });
+        res.cookie('formCookie', '', { secure: true });
```

Comment:
I think the clearCookie command can be used rather than setting it to blank: https://expressjs.com/en/api.html#res.clearCookie